Automattic/wp-calypso
# Stepper: Build bulk domain transfer flow

## Proposed Changes

This adds a working first iteration of bulk-domain transfer flow.

## Testing steps
1. Go to the live link, path /setup/bulk-domain-transfer/domains.
2. Go through the into step.
3. Enter your unlocked domain name and auth code. This won't transfer the domain.
4. Click "Continue".
5. You should land in checkout.
6. The checkout should have all the domains you entered.
